doc/usermanual/po/de.po

```diff
@@ -2473,9 +2475,9 @@ msgid ""
 "a physical copy step. Importing a folder into darktable is therefore "
 "<emphasis>not</emphasis> a backup operation of that folder."
 msgstr ""
-"Zu beachten ist, dass das Importieren von Bildern in darktable kein kopieren "
-"der Bilder bedeutet. Das Importieren eines Ordners in darktable entspricht "
-"daher <emphasis>keinem</emphasis> Backup dieses Ordners."
+"Zu beachten ist, dass das Importieren von Bildern in darktable kein Kopieren "
+"der Bilder importieren. Das Importieren eines Ordners in darktable "
+"entspricht daher <emphasis>keinem</emphasis> Backup dieses Ordners."
```

Comment:
Hier würde ich den Satz wie folgt stehen lassen:
Zu beachten ist, dass das Importieren von Bildern in darktable kein Kopieren der Bilder bedeutet.

Reply:
Hier hab ich wohl einen Fehler ausgebessert und einen neuen hinzugefügt.